firestore/integration_test_internal/src/util/integration_test_util.cc

```diff
@@ -37,7 +37,8 @@ struct TestFriend {
   static FirestoreInternal* CreateTestFirestoreInternal(App* app) {
 #if !defined(__ANDROID__)
     return new FirestoreInternal(
-        app, absl::make_unique<credentials::EmptyAuthCredentialsProvider>(),
+        app, /*database_id=*/"(default)",
```

Comment:
Fill in `(default)` in test firestore, will adjust to take dynamic database_id later while implementing integration tests